variables.tf

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_name" {
+  description = "Project Name"
+  default     = ""
+}
+
+variable "location_code" {
+  description = "A code representing a single character for the type of physical location ('g' for Google cloud, 'm' for Marketo datacenters) plus a short string indicating zone or location (e.x. gusw1, guse1, gase1, etc...). This will be used as the first segment of the machine's hostname"
```

Comment:
This should be genericized, and not be particularly tied to specific user concerns.